src/main/java/eu/dissco/core/translator/terms/BaseDigitalObjectDirector.java

```diff
@@ -197,56 +181,35 @@ public DigitalSpecimen assembleDigitalSpecimenTerm(JsonNode data, boolean dwc)
     ds.withIdentifiers(assembleIdentifiers(data));
     ds.withCitations(assembleSpecimenCitations(data, dwc));
```

Comment:
assembleSpecimenCitations -> gatherCitations, then you can remove the wrapper function